### PR TITLE
OCM-3884 | fix: validate version and HCP default ingress attributes

### PR DESCRIPTION
**Pull request by gdbranco**
Related issue: [OCM-3884](https://issues.redhat.com//browse/OCM-3884)

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -2655,14 +2650,25 @@ func run(cmd *cobra.Command, _ []string) {
 		r.Reporter.Errorf("There was a problem checking version compatibility: %v", err)
 		os.Exit(1)
 	}
+	if ingress.IsDefaultIngressSetViaCLI(cmd.Flags()) {
+		if isHostedCP {
+			r.Reporter.Errorf("Updating default ingress settings is not supported for Hosted Control Plane clusters")
```
[the remark this answers is not in the document]
Comment (lucasponce) [reply cmd/create/cluster/cmd.go:2655]: Day 2 operations on ingress are valid, like changing the visibility.

What is the context to this?

[the remark this answers is not in the document]
Comment (lucasponce) [reply cmd/create/cluster/cmd.go:2655]: Ingress day 2 operations are allowed, so, no, we can't add this validation
image

[the remark this answers is not in the document]
Comment (lucasponce) [reply cmd/create/cluster/cmd.go:2655]: I miss some context, I've edited my previous comments.

Are these changes compatible with defining the visibility of the ingress ?

[the remark this answers is not in the document]
Comment (gdbranco) [reply cmd/create/cluster/cmd.go:2655]: This is just related to day1 operations. HCP has never had day1 ingress attributes as far as I know, so this is only covered for classic. We also block it in the API at the moment

[the remark this answers is not in the document]
Comment (andreadecorte) [reply cmd/create/cluster/cmd.go:2655]: yes I think it should be fine

**Pull request merged**